### PR TITLE
Move the CI from Travis to Circle

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Now that we use to build our CI workflow with CircleCI and that we are fine with it, we've decided to give Fonzie's CI a little refactoring.

## Proposal

- [x] add `.circleci/config.yml`
- [x] remove the node stack
- [x] remove TravisCI-related files

During this refactoring I also add to upgrade the base `edxapp` release to `hawthorn.1` :tada:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Change Log
```
Comment (lunika) [inline CHANGELOG.md]:
```suggestion
## Changelog
```

File: Makefile [hunk 2]
```diff
@@ -40,13 +40,11 @@ help: ## display this help message
 	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
 .PHONY: help
 
-bootstrap: build ## bootstrap the project
-	$(COMPOSE_RUN_LMS) python manage.py lms migrate
+bootstrap: run migrate ## bootstrap the project
```
Comment (lunika) [inline Makefile:43]: you don't want to `build` before running ? You suppose that tunning the application will build it at the first time ?

Comment (jmaupetit) [reply Makefile:43]: Yes it will.

Comment (lunika) [reply Makefile:43]: Ok, for me this is not what I expect from `bootstrap`, but it's ok, keep it like this.

**Review by lunika — APPROVED** [review]

**Review by rmoch — APPROVED** [review]
Seems ok, yet we will miss an nginx container to use Fonzie to serve grade reports with X-accel

**Pull request merged**